source/grid/tria.cc

```diff
@@ -17,7 +17,6 @@
 #include <deal.II/base/memory_consumption.h>
 #include <deal.II/base/table.h>
 #include <deal.II/base/geometry_info.h>
-#include <deal.II/base/std_cxx11/bind.h>
```

Comment:
I had been using std_cxx11::bind, but then second-guessed and went to std::bind. I must have then assumed that I had added this, it has been fixed.